media-libs/gst-plugins-base/gst-plugins-base-1.22.11.ebuild

```diff
@@ -43,7 +43,7 @@ REQUIRED_USE="
 # Dependencies needed by opengl library and plugin (enabled via USE gles2 and/or opengl)
 # dmabuf automagic from libdrm headers (drm_fourcc.h) and EGL, so ensure it with USE=egl (platform independent header used only, thus no MULTILIB_USEDEP); provides dmabuf based upload/download/eglimage options
 GL_DEPS="
-	>=media-libs/mesa-9.0[egl(+)?,gbm(+)?,gles2?,wayland?,${MULTILIB_USEDEP}]
+	>=media-libs/mesa-9.0[egl(+)?,gbm(+)?,gles2(+)?,wayland?,${MULTILIB_USEDEP}]
```

Comment:
I think this might need to be a `opengl` USE dep instead now?

Reply:
Hmm yeah, for some reason I could have sworn it said `-Dgles2=true` last night but I guess I was just fried.